content/tech-blog-archives/getting-started-with-hacking-on-the-list/contents.lr

```diff
@@ -1,6 +1,7 @@
 title: Getting started with hacking on The List
 ---
-author: Matt Lee
+categories:
+Lee
```

Comment:
There's some kind of error in the script. This post should have "development" as a category and nothing else (see: original file: https://raw.githubusercontent.com/creativecommons/labs/gh-pages/_posts/2014-12-15-getting-started-hacking-on-the-list.md)

Reply:
This is the only file with this error. I forgot to add the categories to it after it was merged in #43 and therefore the script not worked properly on it. I will correct it.